testutils/inject/camera.go

```diff
@@ -52,20 +47,6 @@ func (c *Camera) NextPointCloud(ctx context.Context) (pointcloud.PointCloud, err
 	return nil, errors.New("NextPointCloud unimplemented")
 }
 
-// Stream calls the injected Stream or the real version.
-func (c *Camera) Stream(
```

Comment:
Were none of our tests using injected camera Stream?

Reply:
they were removed in a previous pr